Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -42,11 +47,12 @@ For instance, if you want to create a HoloLens application, switch to Universal
 - Click on the **Switch Platform** button
 
 ## Import MRTK packages into your Unity project
-1. Create a new Unity project, or open an existing project. When creating a project, make sure to select "3D" as the template type. 
 
-2. Import the `Microsoft.MixedRealityToolkit.Unity.Foundation.unitypackage` you downloaded by going into "Asset -> Import Package -> Custom Package", selecting the .unitypackage file, ensure all items to import are checked, and then selecting "Import".
-
-3. Import `Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage` following the same steps as above. The examples package is optional but contains useful demonstration scenes for current MRTK features.
+1. Create a new Unity project, or open an existing project. When creating a project, make sure to select "3D" as the template type. 
+1. Import the `Microsoft.MixedRealityToolkit.Unity.Foundation.unitypackage` you downloaded by going into "Asset -> Import Package -> Custom Package", selecting the .unitypackage file, ensure all items to import are checked, and then selecting "Import".
+1. Import the `Microsoft.MixedRealityToolkit.Unity.Extensions.unitypackage` following the same steps as above. The extensions package provides a set of useful optional components for the MRTK.
+1. Import the 'Microsoft.MixedRealityToolkit.Unity.Tools.unitypackage` following the same steps as for the foundation package. These tools package is optional and contains useful tools, such as the ExtensionServiceCreator, that enhance the MRTK developer experience.
+1. Import `Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage` following the same steps as for the foundation package. The examples package is optional and contains useful demonstration scenes for current MRTK features.
```

Comment:
I would put the examples before the other things - I would say things like the hand interaction example scene is more important to get than some of the tools or other extensions (currently, only one extension)

Reply:
will update. thx!